docs/persistence.rst

```diff
@@ -129,8 +129,13 @@ for more details.
 Command Line Interface
 ######################
 
-Skops has a command line interface to convert scikit-learn models persisted with
-``Pickle`` to ``Skops`` files.
+Skops has a command line interface to:
+
+- convert scikit-learn models persisted with ``Pickle`` to ``Skops`` files.
+- update ``Skops`` files to the latest version.
+
+``skops convert``
+-----------------
```

Comment:
The headings of the docs are messed up:

![Screenshot from 2023-05-02 11-32-00](https://user-images.githubusercontent.com/6229650/235631343-f9baa826-3ed1-4e70-ad01-e07f57e75f70.png)

I believe the problem is the use of `----` here. If you look at the top of the file, the title uses `====` (level 1) and the section under the title, "Usage", uses `----` (level 2), and the subsection of "Usage", "Command Line Interface", uses `####` (level 3). Thus, the two subcommands, "skops convert" and "skops update", cannot also use `----` (level 2), they have to use a different type of heading, e.g. `~~~~` (level 4).

I would also be fine to elevate "Command Line Interface" to the same level as "Usage", i.e. to use `----`, but this would still require changing the `----` from the subcommands. Also, "Visualization" below would also need to be elevated (to level 2) in that case.

Yeah, the way rst handles sections is ... not optimal.

Reply:
Thank you for pointing this out! I forgot to check the rendered docs, changed it now :)